pipelines-python/train/publish_to_endpoint.py

```diff
@@ -0,0 +1,40 @@
+import os
+import argparse
+import azureml.core
+from azureml.core import Workspace
+from azureml.pipeline.core import Pipeline, PublishedPipeline, PipelineEndpoint
+
+print("Azure ML SDK version:", azureml.core.VERSION)
+
+parser = argparse.ArgumentParser("publish_to_pipeline_endpoint")
+parser.add_argument("--pipeline_id", type=str, help="Id of the published pipeline that should get added to the Pipeline Endpoint", required=True)
+parser.add_argument("--pipeline_endpoint_name", type=str, help="Name of the Pipeline Endpoint that the the pipeline should be added to", required=True)
+args = parser.parse_args()
+print(f'Arguments: {args}')
+
+print('Connecting to workspace')
+ws = Workspace.from_config()
+print(f'WS name: {ws.name}\nRegion: {ws.location}\nSubscription id: {ws.subscription_id}\nResource group: {ws.resource_group}')
```

Comment:
Why are we connecting to the workspace twice?

Reply:
Oops, this is why we do reviews...! 👍